antarest/study/storage/rawstudy/watcher.py

```diff
@@ -36,6 +36,8 @@
 
 logger = logging.getLogger(__name__)
 
+UPGRADE_TMP_FOLDERS_SUFFIX = ".tmp"
```

Comment:
This is not only for Upgrade but also for timeseries generation so we should find another name.